packages/http-core/test/integration/spec/request.js

```diff
@@ -9,9 +9,9 @@ import {HttpError, request} from '../..';
 import makeLocalUrl from '@ciscospark/test-helper-make-local-url';
 import sinon from '@ciscospark/test-helper-sinon';
 
-describe(`http-core`, function() {
-  this.timeout(30000);
-  describe(`request()`, () => {
+describe(`http-core`, () => {
+  describe(`request()`, function() {
```

Comment:
Actually, that's a mistake. the `function()` should be at the top level. It's because the fat-arrow function binds `this` to the global scope, thus making `this.timeout()` undefined.